package.json

```diff
@@ -123,13 +109,11 @@
     "@graphql-codegen/typescript-react-apollo": "^3.2.5",
     "@playwright/test": "^1.40.0",
     "@saleor/app-sdk": "0.47.2",
-    "@sentry/cli": "^2.33.0",
```

Comment:
Are you sure we don't need Sentry CLI?

Reply:
It may be used by something "under the hood" so I'll keep it just to be safe.